app/k9mail/src/main/java/com/fsck/k9/account/AccountOwnerNameProvider.kt

```diff
@@ -0,0 +1,12 @@
+package com.fsck.k9.account
+
+import app.k9mail.feature.account.setup.domain.ExternalContract
+import com.fsck.k9.Preferences
+
+class AccountOwnerNameProvider(
+    private val preferences: Preferences,
+) : ExternalContract.AccountOwnerNameProvider {
+    override fun getOwnerName(): String? {
+        return preferences.defaultAccount?.senderName
```

Comment:
This call has the potential to perform disk I/O. So we should only call it from a background thread.